src/main/java/seedu/address/logic/parser/modulelistparsers/ViewArchivedModulesParser.java

```diff
@@ -0,0 +1,16 @@
+package seedu.address.logic.parser.modulelistparsers;
+
+import seedu.address.logic.commands.modulelistcommands.ViewArchivedModulesCommand;
+import seedu.address.logic.parser.Parser;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+public class ViewArchivedModulesParser implements Parser<ViewArchivedModulesCommand> {
```

Comment:
If there is no need to parse the user inputs, we can return the command in the ModuleListParser itself, I think this class is redundant.

Reply:
Alright noted thanks.